packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -502,6 +512,51 @@ test.serial('complete auction liquidator gets proceeds', async t => {
   await assertPayouts(t, liqSeat, currency, collateral, 231n, 800n);
 });
 
+// serial because dynamicConfig is shared across tests
```

Comment:
I wish Ava let "serial" be an option on the whole module so this didn't need repeating.

is it worth making these parallel-izable?